bitnami/odoo/values.yaml

```diff
@@ -114,6 +114,9 @@ odooPassword: ""
 ## @param odooSkipInstall Skip Odoo installation wizard
 ##
 odooSkipInstall: false
+## @param odooSkipModulesUpdate Skip Odoo update wizard
+##
+odooSkipModulesUpdate: true
```

Comment:
```suggestion
## @param odooSkipModulesUpdate Skip update of modules during the deployment
##
odooSkipModulesUpdate: no
```

This would change the default behavior. I would not skip this by default

https://github.com/bitnami/containers/blob/main/bitnami/odoo/17/debian-12/rootfs/opt/bitnami/scripts/odoo-env.sh#L96

Reply:
Sorry, my fault. In that case, use "false" instead of "no" but the idea is to follow the same approach we have in the container.